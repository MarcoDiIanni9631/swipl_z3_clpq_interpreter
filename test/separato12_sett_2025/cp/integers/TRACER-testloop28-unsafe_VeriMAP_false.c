# 1 "MAP/SAFE-exbench/TRACER-testloop28.tmp.c"
# 1 "<command-line>"
# 1 "MAP/SAFE-exbench/TRACER-testloop28.tmp.c"
# 20 "MAP/SAFE-exbench/TRACER-testloop28.tmp.c"
//extern int unknown();
main(){
  int i,n,x, NONDET;

  x=0;
  i=0;
  while (i<n) {
    if (unknown() >0){
      __VERIFIER_assert(!( x>0 ));
    }
    else{
      x = 1;
    }
    i++;
  }
}
