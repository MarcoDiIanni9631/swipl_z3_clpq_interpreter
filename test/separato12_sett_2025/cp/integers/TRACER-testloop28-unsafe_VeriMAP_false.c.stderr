MAP/SAFE-exbench/TRACER-testloop28.tmp.c:26: Warning: Body of function main falls-through. Adding a return statement
